README.md

```diff
@@ -30,6 +30,13 @@ find . -name "*_shpec.sh" | entr shpec
 [shpec's own tests](https://github.com/rylnd/shpec/tree/master/shpec/shpec_shpec.sh)
 are a great place to start. More detailed examples to come.
 
+### Mocking
```

Comment:
If waffle.io allows everything we're doing to be visible through Github, I'm fine with trying that out. If not, I'd rather not add another tool; Issues with Labels and Milestones should be enough.